introduction.md

```diff
@@ -6,41 +6,170 @@ layout: documentation
 
 # About openHAB
 
-openHAB **software** integrates different home automation systems, devices and technologies into a single solution.
-It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
+The  **open H**ome  **A**utomation  **B**us (openHAB,  *pronounced ˈəʊpənˈhæb*) is an open source, technology agnostic home automation platform which runs as the center of your smart home! 
 
-## Getting Started
+Some of openHAB's strengths are:
 
-We highly recommend that you read the next chapter titled [Concepts]({{base}}/concepts/index.html).
-It introduces a number of important ideas that will help you as you install and begin to configure openHAB for the first time.
+- Its ability to integrate a multitude of other devices and systems. openHAB includes other home automation systems, (smart) devices and other technologies into a single solution
+- To provide a uniform user interfaces and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved
+- Giving you the most flexible tool available to make almost any home automation wish come true; if you can think it, odds are that you can implement it with openHAB.
 
-openHAB runs on many popular platforms including Linux, Windows and macOS.
-You can find specific installation instructions for these and other platforms in the [Installation Overview]({{base}}/installation/index.html) article.
-Many people find that the simplest way to experiment with openHAB is to get a [Raspberry Pi](https://raspberrypi.org) and install [openHABian]({{base}}/installation/openhabian.html); a "hastle-free openHAB setup".
-While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB home automation system capable of automating your entire home.
+## What you need to know before you start
 
-Once you have openHAB up and running, the [Configuration]({{base}}/configuration/index.html) article contains everything you need to know to get your openHAB installation talking to different devices around your home.
-You can use [Sitemaps]({{base}}/configuration/sitemaps.html) to control how the status of these devices are displayed on different openHAB User Interfaces, and you can begin to experiment with [Rules]({{base}}/configuration/rules-dsl.html) in order to get the most out of your installation.
+**When home automation just seems to work, it is always the result of hard work.** 
+Home automation is fascinating and requires a considerable investment of your time. 
+Here are some key considerations especially for new users. To be successful, you will need to:
+
+* **Start slowly and one step at a time.**
```

Comment:
I am not sure about the bold usage here.
It looks a bit misplaced and all of those points are highlighted anyway through the list.

Reply:
Unbolded, no problem.